components/order_manager.py

```diff
@@ -0,0 +1,20 @@
+from settings_config import firebase_client
+
+"""Class OrderManager is responsible for storing dydx_orders data in firebase cloud storage.
+   It has two method store_order_data() that is responsible for storing data in DB.
+   Method update_order_data() is used to update data in DB. """
+
+
+class OrderManager(object):
+    def __init__(self):
+        self.firebase_client = firebase_client
+
+    def store_order_data(self, order_data):
+        self.firebase_client.collection('dydx_orders').document(order_data["id"]).set(
+            order_data
+        )
+
+    def update_order_data(self, order_id):
+        self.firebase_client.collection('dydx_orders').document(order_id).update(
+            {"status": "Cancel"}
+        )
```

Comment:
hey can you check if there's anyway to track the progress of setting and updating ? 
because if either of them fail, we'll have to catch the error appropriately.

Reply:
m not sure if there is or not i tried  but i don't find any. also the firebase is not that much for error handling